File: chapter07/scalapackage/TestTiger.scala
package com.atguigu.chapter07.scalapackage.hello2

object TestTiger {
  def main(args: Array[String]): Unit = {
    //使用xh的Tiger
    val tiger1 = new com.atguigu.chapter07.scalapackage.xh.Tiger
    //使用xm的Tiger
    val tiger2 = new com.atguigu.chapter07.scalapackage.xm.Tiger
    println(tiger1 + " " + tiger2)

  }
}

class Employee {

}

File: chapter07/javapackage/TestTiger.java
package com.atguigu.chapter07.javapackage;

public class TestTiger {
    public static void main(String[] args) {
        //使用xm的Tiger
        com.atguigu.chapter07.javapackage.xm.Tiger tiger01 = new com.atguigu.chapter07.javapackage.xm.Tiger();
        //使用xh的Tiger
        com.atguigu.chapter07.javapackage.xh.Tiger tiger02 = new com.atguigu.chapter07.javapackage.xh.Tiger();

        System.out.println("tiger01=" + tiger01 + "tiger02=" + tiger02);
    }
}
